src/sinol_make/commands/run/__init__.py

```diff
@@ -747,6 +748,25 @@ def run(self, args):
             example_tests = [test for test in self.tests if self.get_group(test) == 0]
             if len(example_tests) == len(self.tests):
                 print(util.warning('Running only on example tests.'))
+
+            output_tests = glob.glob(os.path.join(os.getcwd(), "out", "*.out"))
```

Comment:
This function is now very long, would it possible to split in into smaller functions?

Reply:
Sure